posthog/exception_utils.py

```diff
@@ -779,6 +779,27 @@ def set_in_app_in_frames(frames, in_app_exclude, in_app_include, project_root=No
     return frames
 
 
+def exception_is_already_captured(error):
+    # type: (Union[BaseException, ExcInfo]) -> bool
+    if isinstance(error, BaseException):
+        return hasattr(error, "__posthog_exception_captured")
+    elif isinstance(error, tuple) and len(error) == 3:
```

Comment:
Not immediately obvious to me why this would be a tuple. Worth adding a comment?